code/modules/clothing/spacesuits/rig/modules/ds13/kinesis/extension.dm

```diff
@@ -13,8 +13,8 @@
 	var/cached_plane
 
 	//The graphical filter we assigned to the held item
-	var/dm_filter/ripple
```

Comment:
Why did you do this?  dm_filter is a valid native type and being defined in this manner is important

Reply:
according to VS Code it's not and it works without it